tests/difference/core/quint_model/README.md

```diff
@@ -0,0 +1,72 @@
+This folder contains a Quint model for the core logic of Cross-Chain Validation (CCV).
+
+### File structure
+The files are as follows:
+- ccv.qnt: Contains the type definitions, data structures, functional logic for CCV.
+The core of the protocol.
+- ccv_statemachine.qnt: Contains the state machine layer for CCV. Very roughly speaking, this could be seen as "e2e tests".
+Also contains invariants.
+- ccv_test.qnt: Contains unit tests for the functional layer of CCV.
+- libararies/*: Libraries that don't belong to CCV, but are used by it.
+
+### Model details
+
+To see the data structures used in the model, see the ProtocolState type in ccv.qnt.
+
+The "public" endpoints of the model are those functions that take as an input the protocol state, and return a Result.
+Other functions are for utility.
+
+The parameters of the protocol are defined as consts in ccv.qnt.
+
+### Tests
+
+To run unit tests, run 
+```
+quint test ccv_test.qnt --main CCVTest
+```
+and 
+```
+quint test ccv_statemachine.qnt --main CCVDefaultStateMachine
+```
+
+### Invariants
+
+The invariants to check are in ccv_statemachine.qnt.
+Check a single invariant by running
+`quint run --invariant INVARIANT_NAME ccv_statemachine.qnt --main CCVDefaultStateMachine`,
```

Comment:
Running something like `quint run --invariant ValidatorUpdatesArePropagated ccv_statemachine.qnt --main CCVDefaultStateMachine` fails similar to the above:
```
Error: Couldn't find any result for id 53 while Inferring effect for ConsumerChains
    at EffectInferrer.fetchResult (/opt/homebrew/lib/node_modules/@informalsystems/quint/dist/src/effects/inferrer.js:262:19)
    at EffectInferrer.effectForName (/opt/homebrew/lib/node_modules/@informalsystems/quint/dist/src/effects/inferrer.js:278:25)
    at EffectInferrer.exitName (/opt/homebrew/lib/node_modules/@informalsystems/quint/dist/src/effects/inferrer.js:99:41)
    at walkExpression (/opt/homebrew/lib/node_modules/@informalsystems/quint/dist/src/IRVisitor.js:272:25)
    at /opt/homebrew/lib/node_modules/@informalsystems/quint/dist/src/IRVisitor.js:289:38
    at Array.forEach (<anonymous>)
    at walkExpression (/opt/homebrew/lib/node_modules/@informalsystems/quint/dist/src/IRVisitor.js:289:23)
    at /opt/homebrew/lib/node_modules/@informalsystems/quint/dist/src/IRVisitor.js:289:38
    at Array.forEach (<anonymous>)
    at walkExpression (/opt/homebrew/lib/node_modules/@informalsystems/quint/dist/src/IRVisitor.js:289:23```

Reply:
Also here, can you make sure Quint is up-to-date?